packages/react-query/src/useSuspenseInfiniteQuery.ts

```diff
@@ -135,14 +135,6 @@ export function useSuspenseInfiniteQuery<
 ): UseSuspenseInfiniteQueryResultOnSuccess<TData> | UseSuspenseInfiniteQueryResultOnLoading
 
 // arg1: options
```

Comment:
Is this a necessary comment?

Reply:
Only for develop library, not user. This comment is only for me or other maintainer to differ 3 sections of overrode function types(arg1, arg2?, arg3?), but this comment won't be contained